appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java

```diff
@@ -83,25 +83,27 @@
 @RunLevel(StartupRunLevel.VAL)
 public class MetricsService implements EventListener {
 
-    private static final Logger LOGGER = Logger.getLogger(MetricsService.class.getName());
-
     @Inject
     Events events;
 
     @Inject
     private ServerEnvironment serverEnv;
-
-    private final Map<String, Boolean> enabledMap;
     
+    @Inject
+    ServiceLocator habitat;
```

Comment:
Perhaps start calling "habitat" "serviceLocator"? It seems "habitat" is the old name that wasn't refactored when "ServiceLocator" was introduced.